PLEASE_READ_FIRST.md

```diff
@@ -0,0 +1,52 @@
+# Introduction
+
+Hi there 👋,
+
+Thank you for completing this brief take-home assignment. Your time and effort are greatly appreciated. Please timebox your effort to a maximum of 3 hours.
+
+The purpose of this exercise is to evaluate your backend skills in Node.js, SQL, and testing. Please note that you may make assumptions, simplifications, or other changes to the problems, but please state them clearly in your write-up when you submit this assignment. Please feel free to use libraries as appropriate, as there is no need to reinvent the wheel.
+
+Before starting, please review the instructions carefully.
+
+**Good luck 🙃**
+
+# Acceptance criteria
+
+Your task is to build a REST API usingSQLite to manage a property list.
+
+The API should enable users perform CRUD actions (create, read, update, delete) on property data from the database.
+
+Requests should be validated where applicable.
+
+Invalid requests and other errors should be handled appropriately along with the appropriate HTTP status codes. You may use your discretion regarding error responses themselves.
+
+Where applicable, endpoints returning multiple records should be filterable and support pagination. You may choose any pagination strategy you are familiar with.
```

Comment:
I'm still thinking we don't require filtering and that is asked to be added in the pair programming interview. Thoughts @a88zach @xdc0 @dbaq